browser_tests/interaction.spec.ts

```diff
@@ -12,9 +12,19 @@ test.describe('Node Interaction', () => {
   })
 
   test.describe('Node Selection', () => {
-    test.afterEach(async ({ comfyPage }) => {
-      // Deselect all nodes
-      await comfyPage.clickEmptySpace()
```

Comment:
This can be removed because nodes' `is_selected` is not serialized and selection is not otherwise persisted across tests.